src/llvm/lib/Target/MSP430/MSP430InstrInfo.td

```diff
@@ -215,7 +215,8 @@ let isCodeGenOnly = 1, usesCustomInserter = 1 in {
   def Srl16    : Pseudo<(outs GR16:$dst), (ins GR16:$src, GR8:$cnt),
                         "# Srl16 PSEUDO",
                         [(set GR16:$dst, (srl GR16:$src, GR8:$cnt))]>;
-  def Clrc     : Pseudo<(outs), (ins), "", [(MSP430clrc)]>;
+  def Rrl      : Pseudo<(outs GR16:$dst), (ins GR16:$src), "",
```

Comment:
Should we indicate the we implicitly use / def status register here?

Reply:
`let Def = [SR]` is here already. And we don't use SR, just clear carry bit.